backend/app/core/crypto.py

```
@@ -0,0 +1,137 @@
+
+        # Pad the data to 16-byte boundary (AES block size)
+        padder = padding.PKCS7(128).padder()
+        padded_data = padder.update(plain_token.encode('utf-8')) + padder.finalize()
+
+        # Encrypt the data
+        encrypted_bytes = encryptor.update(padded_data) + encryptor.finalize()
+
+        # Return base64 encoded encrypted data
+        return base64.b64encode(encrypted_bytes).decode('utf-8')
+    except Exception as e:
+        logger.error(f"Failed to encrypt git token: {str(e)}")
```

Ruff (0.14.4)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)